checkov/sca_package_2/output.py

```diff
@@ -17,6 +17,8 @@
 from checkov.common.typing import _LicenseStatusWithLines
 from checkov.common.output.common import compare_table_items_severity
 
+InterestingRiskFactors = ["IsUsed"]
```

Comment:
The naming convention for constants we use is all-caps snake case.
Also, let's rename this to indicate it is related to reachability risk factors.

Something like `REACHABILITY_RISK_FACTORS_KEYS`